gbp/config.py

```diff
@@ -201,6 +202,11 @@ class GbpOptionParser(OptionParser):
         'upstream-tree':
             "Where to generate the upstream tarball from "
             "(tag or branch), default is '%(upstream-tree)s'",
+        'upstream-remote':
+            "If debian/upstream/metadata specifies an upstream git repository, "
+            "the remote name to fetch it into, default is "
+            "'%(upstream-remote)s'. Set to an empty string to disable cloning "
+            "upstream",
```

Comment:
Instead of `Set to an empty string to disable cloning "upstream"` it'd rather do a `default is ...` as with most of the other options.